.pipelines/deploy-azure-env-tag.yml

```
@@ -0,0 +1,31 @@
+# Azure DevOps Pipeline deploying the azure hosted environment
+trigger: none
+pr: none
+
+parameters:
+- name: location
+- name: vsoConfigBuildID
+- name: aroImageTag
+- name: rpImageAcr
+  type: string
+  default: arosvc
```

```suggestion
  - name: aroImageTag
  - name: rpImageAcr
    type: string
    default: arosvc
```
New parameters added to controll the deployment